xikolo-common/UserDefaultsKeys.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  UserDefaultsKeys.swift
+//  xikolo-ios
+//
+//  Created by Max Bothe on 26.07.17.
+//  Copyright © 2017 HPI. All rights reserved.
+//
+
+import Foundation
+
+struct UserDefaultsKeys {
+
+    // negative form to have content preload activated by default
+    // since UserDefault will return `false` if no value for the key is present
+    static let noContentPreloadKey = "de.xikolo.ios.content.noPreload"
+
+    static let playbackRateKey = "de.xikolo.ios.video.playbackRate"
```

Comment:
Oh and if you want to store the keys here, please at least make it a complete list. I used to store them [here](https://github.com/openHPI/xikolo-ios/blob/0d3508197fe11975ead2c629424c6e995ea165f9/xikolo-common/Data/Helper/UserProfileHelper.swift#L19)